arctic/chunkstore/_chunker.py

```diff
@@ -60,3 +60,13 @@ def exclude(self, data, range_obj):
         data, filtered by range_obj
         """
         raise NotImplementedError
+
+    def chunk_to_str(self, chunk_id):
+        """
+        Converts a chunk identifier to a string
```

Comment:
sounds good. its basically there to make sure the parts of the chunk range convertable to strings (for a date chunker, they are datetimes)